botocore/credentials.py

```diff
@@ -16,6 +16,10 @@
 import logging
 import os
 import getpass
+import re
+import xml.etree.ElementTree as ET
+import base64
+import json
```

Comment:
Turns out my current implementation does not need to use json module at all. I'll remove that in next commit.